### PR TITLE
Patching watcher commands and cavity display test errors

### PR DESCRIPTION
**Pull request by lisazacarias**
The watcher show command was launching as physics instead of laci and so wasn't finding the session.

Separately, lazy cavity display PV creation was causing problems in the backend launched from the watcher buttons but undoing it was causing problems in the testing. This is an attempt to make them lazy for testing but not lazy in production (plus general testing cleanup while trying to find the problem)

### CODE REVIEW

File: displays/cavity_display/backend/backend_cavity.py [hunk 1]
```diff
@@ -1,39 +1,56 @@
...
-from epics import caput
+from lcls_tools.common.controls.pyepics.utils import PV
 
 from displays.cavity_display.backend.fault import Fault, FaultCounter, PVInvalidError
+from displays.cavity_display.utils import utils
 from displays.cavity_display.utils.utils import (
     STATUS_SUFFIX,
     DESCRIPTION_SUFFIX,
     SEVERITY_SUFFIX,
-    parse_csv,
```
Comment (Derikka) [inline displays/cavity_display/backend/backend_cavity.py:12]: Is there any particular reason we took `parse_csv` out of this import statement?

File: displays/cavity_display/backend/backend_cavity.py [hunk 2]
```diff
@@ -1,39 +1,56 @@
 from collections import OrderedDict, defaultdict
 from datetime import datetime
-from typing import DefaultDict
+from typing import DefaultDict, Optional
 
-from epics import caput
+from lcls_tools.common.controls.pyepics.utils import PV
 
 from displays.cavity_display.backend.fault import Fault, FaultCounter, PVInvalidError
+from displays.cavity_display.utils import utils
```
Comment (Derikka) [inline displays/cavity_display/backend/backend_cavity.py:8]: Is there any particular reason we took `parse_csv` out of the import statement below and added in this line?

File: displays/cavity_display/backend/fault.py [hunk 3]
```diff
@@ -73,12 +72,13 @@ def __init__(
...
         # Storing PV name as a string instead of making a PV obj
         self.pv: str = pv
-        self._pv_obj: Optional[PV] = None
+        # TODO figure out why lazy generation breaks the backend runner
+        self._pv_obj: Optional[PV] = PV(self.pv) if not lazy_pv else None
 
     @property
     def pv_obj(self) -> PV:
         if not self._pv_obj:
-            self._pv_obj = PV(self.pv, connection_timeout=PV_TIMEOUT)
```
Comment (Derikka) [inline displays/cavity_display/backend/fault.py:81]: Why don't we need the connection_time anymore for PVs? I noticed we decided not to include a connection_timeout for all the new PV() things in this pull request

File: displays/cavity_display/backend/backend_cavity.py (hunk 2 above)
Comment (lisazacarias) [reply displays/cavity_display/backend/backend_cavity.py:8]: Yeah, for patching! The patch doesn't work if you import it directly for a reason I didn't really bother to research that thoroughly lol

File: displays/cavity_display/backend/fault.py (hunk 3 above)
Comment (lisazacarias) [reply displays/cavity_display/backend/fault.py:81]: I realized it was redundant; the default timeout time is already that value lol

Comment (Derikka) [reply displays/cavity_display/backend/fault.py:81]: Oh, nice, that's convenient!

File: displays/cavity_display/backend/backend_cavity.py [hunk 4]
```diff
@@ -1,39 +1,56 @@
...
+        self._status_pv_obj: Optional[PV] = None
         self.severity_pv: str = self.pv_addr(SEVERITY_SUFFIX)
+        self._severity_pv_obj: Optional[PV] = None
         self.description_pv: str = self.pv_addr(DESCRIPTION_SUFFIX)
+        self._description_pv_obj: Optional[PV] = None
 
         self.faults: OrderedDict[int, Fault] = OrderedDict()
         self.create_faults()
 
+    @property
```
[the remark this answers is not in the document]
Comment (lisazacarias) [reply displays/cavity_display/backend/backend_cavity.py:34]: Connection failures how? I think the way I've tried to do it is to push the error handling into the functions that call the pv objects in try except blocks. Is that what you mean?

[the remark this answers is not in the document]
Comment (hmarts9) [reply displays/cavity_display/backend/backend_cavity.py:34]: Yeah that's what I meant but oh yeah no you're right I should've looked more carefully its in the run_through_faults method lol.

**Pull request merged**